### DISCUSSION TITLE
Does Docker-compose install require cron?

### DISCUSSION THREAD

**Discussion by eng3** [Taxel/PlexTraktSync#837] [Q&A] [answered]
When I run the docker container manually, the sync works fine.  I've added it to docker-compose.  Once I run "docker-compose up -d" will it continue to sync on its own?  or do I need to create a cron job?

**Comment by stevietv** [comment, marked as answer]
its described here. https://github.com/Taxel/PlexTraktSync#docker-compose

**Reply by eng3** [reply]
AH,  I see.  Sorry, I missed the part where you defined what the "watch" command does.  For some reason I assumed it was for watching the logs or something.  Thanks
